.goreleaser.yaml

```diff
@@ -25,9 +25,6 @@ builds:
 archives:
   - replacements:
       darwin: macOS
-      linux: Linux
-      windows: Windows
-      amd64: x86_64
```

Comment:
Was there a reason you changed then naming scheme here?

The change to call this `x86_64` was deliberate, as that is definitely the dominant term on macOS. 
On windows it seems to be mostly `x64` with a bit of `x86_64` thrown in there. 

It's really only linux that prefers `amd64`, which is a bit of a niche, especially given that we expect most linuxy people to go via the deb or rpm anyway.

Reply:
Blocking on this

Reply:
I asked the question a while back if there was a particular reason for this replacement and was told there wasn't. I also looked at the GitHub CLI (as we've been using it as a base line for certain things) and they do not have this replacement.

Reply:
🤷 ok, I'm good with that.